SU-SWS/stanford_profile
# D8CORE-1401: wysiwyg tweaks

# READY FOR REVIEW

# Summary
- fixes issues with WYSIWYG styles

# Needed By (Date)
- Soon

# Urgency
- Fixes broken stuff in the WYSIWYG

# Steps to Test

1. Pull this branch
2. clear caches
3. verify styles in the WYSIWYG:
```
p {
  padding-bottom: 3.4rem; 
  margin-bottom: 0px; 
} 
And for the H2 + H3:

h2, h3 {
  margin-bottom: 0; /*as in the modular spacer*/
}
Callout:

.su-callout-text {
  margin-top: 0; /*as in the modular spacer*/
}
```
Additionally, 
Verify that the spacing is correct for the "related" or "Card" WYSIWYG style.
It fixes the padding when this style is first:

![Screen Shot 2020-03-06 at 4 33 11 PM](https://user-images.githubusercontent.com/284440/76132550-5dcc7600-5fc8-11ea-882c-3d2c47c25b27.png)
 Versus:

![Screen Shot 2020-03-06 at 4 34 48 PM](https://user-images.githubusercontent.com/284440/76132599-910f0500-5fc8-11ea-99a9-b047cf106901.png)

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-1401
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)
@kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
